modules/howtos/pages/subdocument-operations.adoc

```diff
@@ -261,50 +271,74 @@ Consider:
 
 Looking at the `some_field` field (which is really `level_0.level_1.level_2.level_3.some_field`), its _immediate_ parent is `level_3`.
 If we were to attempt to insert another field, `level_0.level_1.level_2.level_3.another_field`, it would succeed because the immediate parent is present.
-However if we were to attempt to _subdoc-insert_ to `level_1.level_2.foo.bar` it would fail, because `level_1.level_2.foo` (which would be the immediate parent) does not exist.
+However if we were to attempt to _insert_ to `level_1.level_2.foo.bar` it would fail, because `level_1.level_2.foo` (which would be the immediate parent) does not exist.
 Attempting to perform such an operation would result in a Path Not Found error.
 
 By default the automatic creation of parents is disabled, as a simple typo in application code can result in a rather confusing document structure.
 Sometimes it is necessary to have the server create the hierarchy however.
-In this case, the _create-path_ option may be used.
+In this case, the _create_path_ option may be used.
 
 [source,python]
 ----
 include::example$subdocument_ops.py[tag=createparents]
 ----
 
-== CAS Semantics
+== Concurrent Modifications
 
-Subdoc mostly eliminates the need for tracking the xref:concurrent-mutations-cluster.adoc[CAS] value.
-Subdoc operations are atomic and therefore if two different threads access two different sub-documents then no conflict will arise.
+Concurrent Sub-Document operations on different parts of a document will not conflict.
 For example the following two blocks can execute concurrently without any risk of conflict:
 
 [source,python]
 ----
 include::example$subdocument_ops.py[tag=cas1]
 ----
 
+Even when modifying the _same_ part of the document, operations will not necessarily conflict.
+For example, two concurrent _arrayappend_ operations to the same array will both succeed, never overwriting the other.
+
+
+So in some cases the application will not need to supply a xref:concurrent-mutations-cluster.adoc[CAS] value to protect
```

Comment:
Does **concurrent-mutations-cluster.adoc** exist? Couldn't find it on my local clone.

Reply:
No, will submit a PR after this to resolve.